components/host-dashboard/constants.js

```diff
@@ -1,5 +1,5 @@
 export const HOST_SECTIONS = {
-  EXPENSES: 'expenses',
+  HOST_EXPENSES: 'host-expenses',
```

Comment:
This breaks the existing admin panel. It displays the Expense page as "Conversations" for some reason:
![image](https://user-images.githubusercontent.com/2119706/230318689-247072e0-924a-4815-9d62-9ee23894710e.png)

Reply:
Ah, good catch! This was unintentional, did not consider that this component was outside of the forked /admin-panel folder

Reply:
Now fixed.